packages/ic-error-types/src/lib.rs

```diff
@@ -478,7 +478,9 @@ mod str_traits {
             }
 
             const ELLIPSIS: &str = "...";
-            assert!(max_len >= ELLIPSIS.len());
+            if max_len <= ELLIPSIS.len() {
+                return ".".repeat(max_len);
+            }
```

Comment:
It seems there is a copy of the `StrEllipsize` trait and implementation in `ic-error-types` -- I suppose the expected action on changing the original type is to simply copy the changes over?

Reply:
I couldn't find any tests here. Am I overlooking them?

Reply:
ah, right, I was assuming there were somewhere here.